meson.build

```diff
@@ -1,27 +1,42 @@
-project(
-    'io.elementary.terminal',
-    'vala', 'c',
-    version: '6.1.1'
-)
+project('io.elementary.terminal', 'vala', 'c', version: '6.1.1', meson_version: '>=0.58')
```

Comment:
I think this is less legible than having arguments separated into their own lines, personally